src/mscorlib/corefx/System/Globalization/JapaneseCalendar.Win32.cs

```diff
@@ -7,12 +7,51 @@ namespace System.Globalization
 {
     public partial class JapaneseCalendar : Calendar
     {
-        public static int GetJapaneseEraCount()
+        private static EraInfo[] GetJapaneseEras()
         {
-            return WinRTInterop.Callbacks.GetJapaneseEraCount();
+            int erasCount = WinRTInterop.Callbacks.GetJapaneseEraCount();
```

Comment:
Are we OK with the Globalization code being duplicated in the CoreCLR repo?
https://github.com/dotnet/coreclr/tree/master/src/mscorlib/src/System/Globalization
and
https://github.com/dotnet/coreclr/tree/master/src/mscorlib/corefx/System/Globalization

What is our long term plan here?